### PR TITLE
feat(route/tableau): Add route for Tableau viz of the day.

### PR DESCRIPTION
**Pull request by KaiyoungYu**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/tableau/viz-of-the-day
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Documentation / 文档说明
  - [ ] EN / 英文文档
  - [ ] CN / 中文文档
- [ ] Full text / 全文获取
  - [ ] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制 
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施? 
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: website/docs/routes/study.md [hunk 1]
```diff
@@ -284,6 +284,11 @@ paramsDesc={['一个整数，筛选最近的limit场比赛，默认为5']}>
 
 <Route author="nczitzk" example="/researchgate/publications/Somsak-Panha" path="/researchgate/publications/:username" paramsDesc={['Username, can be found in URL']} puppeteer="1" anticrawler="1"/>
 
+## Tableau {#tableau}
+
+### Viz of the day {#tableau-viz-of-the-day}
+<Route author="KaiyoungYu" example="/tableau/viz-of-the-day" path="/tableau/viz-of-the-day" radar="1" />
```
Comment (TonyRL) [inline website/docs/routes/study.md:290]:
```suggestion
### Viz of the day {#tableau-viz-of-the-day}

<Route author="KaiyoungYu" example="/tableau/viz-of-the-day" path="/tableau/viz-of-the-day" radar="1" />
```

File: lib/v2/tableau/viz-of-the-day.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    // Your logic here
+    const rootUrl = 'https://public.tableau.com/api/gallery?page=0&count=20&galleryType=viz-of-the-day';
+    const { data: response } = await got(rootUrl);
+
+    const items = response.items.map((item) => ({
+        title: item.title,
+        link: item.sourceUrl,
+        pubDate: parseDate(item.galleryItemPublicationDate),
+        author: item.authorName,
+        description: item.description,
```
Comment (TonyRL) [inline lib/v2/tableau/viz-of-the-day.js:15]: Consider adding `screenshot` to the description as well.

File: website/docs/routes/study.md (hunk 1 above)
Comment (KaiyoungYu) [reply website/docs/routes/study.md]: tks, I modified it.

File: lib/v2/tableau/viz-of-the-day.js (hunk 2 above)
Comment (KaiyoungYu) [reply lib/v2/tableau/viz-of-the-day.js]: tks, I figured out how to add new html into description, and it is done

**Pull request merged**